data/images/recordsheets/naval_dualturret_standard.svg

```diff
@@ -0,0 +1,1495 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I normally wouldn't review these by hand, but the files are formatted quite bizarrely.

Reply:
Yes, they're built in memory using DOM then written to a file using Batik's SVGGraphics2D#stream. I haven't found a way to influence the formatting of the output, but I haven't spent a lot of time looking.